packages/oc-template-react-compiler/lib/higherOrderServerTemplate.js

```diff
@@ -16,7 +16,8 @@ export const data = (context, callback) => {
       _componentName: "${componentName}",
       _componentVersion: "${componentVersion}"
     });
-    const srcPath = (context.env && context.env.name === "local") ? context.staticPath : "https:" + context.staticPath ;
+    const srcPathHasProtocol = context.staticPath.indexOf("http") === 0;
+    const srcPath = srcPathHasProtocol ? context.staticPath : ("https:" + context.staticPath);
```

Comment:
I think this is consistent with what we did on the OC side, instead of relying on the context env to be set to local